templates/htmx-survey.html

```diff
@@ -17,8 +17,10 @@
                 hx-post="/surveys/submit-survey/{{ survey_id }}"
                 hx-include="[name='question']"
                 hx-swap="innerHTML"
-                _="on click toggle .hidden on #survey">
+                _="on click toggle .hidden on #survey
+                and window.location.reload()">
                 {{_('survey_submit')}}</button>
```

Comment:
> could you please make all texts in this component as `{{ _('TRANSLATION_KEY') }}`; i.e., description, question and others. You don't have to return them then from your `class_survey` function.

I don't think that would be possible because the questions have to be generated based on which ones are unanswered and besides that it won't be generic if I would do that because it would have to load specifically the class survey questions and description

Reply:
> I don't think that would be possible because the questions have to be generated based on which ones are unanswered and besides that it won't be generic if I would do that because it would have to load specifically the class survey questions and description

What is not possible? I see that the `class_survey` just returns:
```python
description = gettext("class_survey_description")
survey_later = gettext("class_survey_later")
```
and these have nothing to do with how you retrieve the questions!

Reply:
`class_survey` is a specific function for generating a class survey. If I add `{{ _('class_survey_description') }}` and `{{ _('class_survey_later') }}` to htmx-survey.html , it would become a survey specifically for class questions instead of a generic survey that can be used anywhere.

Reply:
Ahha okay let's keep it as is then. Do we then need the _() rather than just {{quetion}}?

Reply:
Btw, perhaps show a cancel button near the submit when click is true.

Reply:
> Ahha i agree, okay let's keep it as is then. Do we then need the _() rather than just {{quetion}}?

I don't understand, where?